packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -211,6 +211,17 @@ export class CdkErrorMapper {
       errorName: 'GetLambdaLayerVersionError',
       classification: 'ERROR',
     },
+    {
+      //This has some overlap with "User:__ is not authorized to perform:__ on resource: __" - some resources cannot be deleted due to lack of permissions
+      errorRegex:
+        /The stack named (?<stackName>.*) is in a failed state. You may need to delete it from the AWS console : DELETE_FAILED \(The following resource\(s\) failed to delete: (?<resources>.*). \)/,
+      humanReadableErrorMessage:
+        'The CloudFormation deletion failed due to {stackName} being in DELETE_FAILED state. Ensure all your resources are able to be deleted',
+      resolutionMessage:
+        'The following resource(s) failed to delete: {resources}. Ensure they are in a state where they can be deleted. Find more information in the CloudFormation AWS Console for this stack.',
```

Comment:
The current message that is produced by the fault has both the resources that caused the failed deletion as well as how to get those resources into a state where they can be deleted, would like to find a way to get that information into the resolution message.